packages/quantic/force-app/main/default/lwc/quanticSearchInterface/quanticSearchInterface.js

```diff
@@ -89,7 +91,9 @@ export default class QuanticSearchInterface extends LightningElement {
         return;
       }
       window.localStorage.removeItem(STANDALONE_SEARCH_BOX_STORAGE_KEY);
-      const {analytics} = JSON.parse(redirectData);
+      const {value, analytics} = JSON.parse(redirectData);
+      
+      engine.dispatch(updateQuery({q: value}));
```

Comment:
This is needed now because the SSB and the SB don't share the engine?

Reply:
indeed. Like the analytics, the query is passed through localstorage.